using-timescaledb/data-retention.md

```diff
@@ -20,16 +20,38 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
-### Automatic Data Retention Policies
+### Automatic Data Retention Policies :enterprise_function:
 
-The `drop_chunks` command can be combined with an external tool for
-job scheduling, like `crontab` or `systemd`, to implement automatic
-data retention policies. Below we give some examples of how to
-implement such policies.
+TimescaleDB includes a background job scheduling framework for automating data
+management tasks, such as enabling easy data retention policies.
 
-Note that the path to `psql` and the parameters to connect to the
-database might differ depending on setup and need to be modified
-accordingly in the examples below.
+To add such data retention policies, a database administrator can create,
+remove, or alter policies that schedule `drop_chunks` to take place on a
+hypertable on a regularly scheduled interval.
```

Comment:
The wording of this is confusing for the command below.  The reader may question whether the interval parameter in the command is for when the `drop_chunks` command runs, or for the data age to apply `drop_chunks` to.

Reply:
Updated.